### PR TITLE
Add bucket stat consistency tc to pacific

### PR DESCRIPTION
**Pull request by ckulal**
Signed-off-by: ckulal <ckulal@redhat.com>

Adding bucket stat consistency check testcase to pacific
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-KB6HCA/S3CMD_tests_1.log

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by udaysk23 — DISMISSED** [review]

**Review by psathyan — COMMENTED** [review]
Looks good to me...

A request, could you rename the suite to `tier-2_rgw_test-using-s3cmd.yaml`? This would help align with the naming guidelines put in place.

File: suites/pacific/rgw/tier-2_rgw_test-using-s3cmd.yaml [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# RHCS 5.0 sanity test suite for RGW daemon with S3CMD.
```
Comment (psathyan) [inline suites/pacific/rgw/tier-2_rgw_test-using-s3cmd.yaml:1]: Please fix the docString

File: suites/pacific/rgw/tier-2_rgw_test-using-s3cmd.yaml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                registry-url: registry.redhat.io
```
Comment (psathyan) [inline suites/pacific/rgw/tier-2_rgw_test-using-s3cmd.yaml:60]: could you remove this configuration?

Comment (ckulal) [reply suites/pacific/rgw/tier-2_rgw_test-using-s3cmd.yaml]: done

**Review by anrao19 — APPROVED** [review]

**Review by psathyan — APPROVED** [review]

**Pull request merged**